pkg/gen/input/workflows/internal/file/update_chart.yaml.template

```diff
@@ -0,0 +1,99 @@
+{{{{ .Header }}}}
+name: Create Update PR
+on:
+  push:
+    branches:
+      - 'main#update'
+      - 'master#update'
```

Comment:
unsure if branch name `update` is too generic. I would suggest `update_chart` as an improvement.

Reply:
I went for `update-chart`, following the make file target